benchresult segmentation fault

Hi, When I run "go run benchmark/benchresult/main.go result" I get below error: I have tried various results file which belongs to benchmark/benchmain/main.go.

Could you please let me know how to fix below error as well as how can I view cpuProf and memProf results of benchmain option for -cpuProfile and -memProfile?
What version of gRPC are you using?
Version = "1.30.0-dev"

What version of Go are you using (go version)?
go version go1.14.1 linux/amd64

What operating system (Linux, Windows, …) and version?
Linux

**ERROR:**
Shared features:
--------------------
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x80a111]

goroutine 1 [running]:
google.golang.org/grpc/benchmark/stats.(*PayloadCurve).Hash(...)
        /root/go/src/google.golang.org/grpc/benchmark/stats/curve.go:174
google.golang.org/grpc/benchmark/stats.Features.partialString(0xc0000253d0, 0x5, 0x0, 0x2540be400, 0x0, 0x0, 0x0, 0x0, 0x1, 0x1, ...)
        /root/go/src/google.golang.org/grpc/benchmark/stats/stats.go:183 +0x7d1
google.golang.org/grpc/benchmark/stats.Features.SharedFeatures(0xc0000253d0, 0x5, 0x0, 0x2540be400, 0x0, 0x0, 0x0, 0x0, 0x1, 0x1, ...)
        /root/go/src/google.golang.org/grpc/benchmark/stats/stats.go:149 +0x1b5
main.formatBenchmark(0x7ffe85bf567e, 0x6)
        /root/go/src/google.golang.org/grpc/benchmark/benchresult/main.go:130 +0x290
main.main()
        /root/go/src/google.golang.org/grpc/benchmark/benchresult/main.go:146 +0x89
exit status 2

--- a/benchmark/stats/stats.go
+++ b/benchmark/stats/stats.go
@@ -180,9 +180,13 @@ func (f Features) partialString(b *bytes.Buffer, wantFeatures []bool, sep, delim
 			case RespSizeBytesIndex:
 				b.WriteString(fmt.Sprintf("RespSize%v%vB%v", sep, f.RespSizeBytes, delim))
 			case ReqPayloadCurveIndex:
-				b.WriteString(fmt.Sprintf("ReqPayloadCurve%vSHA-256:%v%v", sep, f.ReqPayloadCurve.Hash(), delim))
+				if f.ReqPayloadCurve != nil {
+					b.WriteString(fmt.Sprintf("ReqPayloadCurve%vSHA-256:%v%v", sep, f.ReqPayloadCurve.Hash(), delim))
+				}
 			case RespPayloadCurveIndex:
-				b.WriteString(fmt.Sprintf("RespPayloadCurve%vSHA-256:%v%v", sep, f.RespPayloadCurve.Hash(), delim))
+				if f.RespPayloadCurve != nil {
+					b.WriteString(fmt.Sprintf("RespPayloadCurve%vSHA-256:%v%v", sep, f.RespPayloadCurve.Hash(), delim))
+				}
 			case CompModesIndex:
 				b.WriteString(fmt.Sprintf("Compressor%v%v%v", sep, f.ModeCompressor, delim))
 			case EnableChannelzIndex:

